app/models/block/hero.rb

```diff
@@ -0,0 +1,21 @@
+module Block
+  HeroImageSources = Data.define(:desktop, :desktop_2x, :tablet, :tablet_2x, :mobile, :mobile_2x)
```

Comment:
What happens if not all of the image sources are provided, or do you think "we" will be expected to create them?

I'm wondering what happens to the page if the images don't exist.

Reply:
I was imagining them as kind of mandatory. At the moment if you leave them out of the YAML, this code will raise and error like:

```
ArgumentError
missing keywords: :desktop, :desktop_2x, :tablet, :tablet_2x, :mobile, :mobile_2x
```

Reply:
Ok. I think this will need to be captured somewhere. Perhaps in some yet-to-be-written docs on adding blocks to pages.

Reply:
Ideally we'd add some exception handling, but that can wait until post-launch.